Updates to TeamsCallingPolicy (#10937)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/New-CsTeamsCallingPolicy.md

```

### -AllowCallRedirect
Setting this parameter provides the ability to configure call redirection capabilities on Teams Phones. 

> Valid options are:
> - Enabled: Users will have the ability to redirect received calls.
> - Disabled: User will not have such ability.
> - UserOverride: This option is not available for use. There's no user interface implemented for its management.

Reviewer: ```suggestion
Valid options are:
- Enabled: Users will have the ability to redirect received calls.
- Disabled: Users will not have such an ability.
- UserOverride: This option is not available for use. There's no user interface implemented for its management.

```

Accept wildcard characters: False
```

### -Identity
Name of the policy instance being created.

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: False
Position: 1
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -InboundFederatedCallRoutingTreatment
Setting this parameter lets you control how inbound federated calls should be routed. 

Valid options are:
- RegularIncoming: No changes are made to default inbound routing. This is the default setting.

> - Unanswered: The inbound federated call will be routed according to the called user unanswered call settings and the call will not be presented to the called user. The called user will see a missed call notification. If the called user have not enabled unanswered call settings the call will be disconnected.

Reviewer: ```suggestion
- Unanswered: The inbound federated call will be routed according to the called user's unanswered call settings and the call will not be presented to the called user. The called user will see a missed call notification. If the called user has not enabled unanswered call settings the call will be disconnected.

```

- Voicemail: The inbound federated call will be routed directly to the called user's voicemail and the call will not be presented to the user. If the called user does not have voicemail enabled the call will be disconnected.

> Setting this parameter to Unanswered or Voicemail will have precedence over other call forwarding settings like call forward/simultaneous ringing to delegate, call groups or call forwarding.

Reviewer: ```suggestion
Setting this parameter to Unanswered or Voicemail will have precedence over other call forwarding settings like call forward/simultaneous ringing to delegate, call groups, or call forwarding.

```

```yaml
Type: String
Parameter Sets: (All)
Aliases:
Applicable: Microsoft Teams

Required: False
Position: Named
Default value: RegularIncoming
Accept pipeline input: False
Accept wildcard characters: False
```

### -InboundPstnCallRoutingTreatment
Setting this parameter lets you control how inbound PSTN calls should be routed. 

Valid options are:
- RegularIncoming: No changes are made to default inbound routing. This is the default setting.

> - Unanswered: The inbound PSTN call will be routed according to the called user unanswered call settings and the call will not be presented to the called user. The called user will see a missed call notification. If the called user have not enabled unanswered call settings the call will be disconnected.

Reviewer: ```suggestion
- Unanswered: The inbound PSTN call will be routed according to the called user's unanswered call settings and the call will not be presented to the called user. The called user will see a missed call notification. If the called user has not enabled unanswered call settings the call will be disconnected.

```

- Voicemail: The inbound PSTN call will be routed directly to the called user's voicemail and the call will not be presented to the user. If the called user does not have voicemail enabled the call will be disconnected.
- UserOverride: For now, setting the value to UserOverride is the same as RegularIncoming.

> Setting this parameter to Unanswered or Voicemail will have precedence over other call forwarding settings like call forward/simultaneous ringing to delegate, call groups or call forwarding.

Reviewer: ```suggestion
Setting this parameter to Unanswered or Voicemail will have precedence over other call forwarding settings like call forward/simultaneous ringing to delegate, call groups, or call forwarding.

```

```yaml
Type: String
Parameter Sets: (All)
Aliases:
Applicable: Microsoft Teams

Required: False
Position: Named
Default value: RegularIncoming
Accept pipeline input: False
Accept wildcard characters: False
```

### -LiveCaptionsEnabledTypeForCalling
Determines whether real-time captions are available for the user in Teams calls.

Valid options are:

> - DisabledUserOverride: Allow the user to turn on live captions.

Reviewer: ```suggestion
- DisabledUserOverride: Allows the user to turn on live captions.

```
